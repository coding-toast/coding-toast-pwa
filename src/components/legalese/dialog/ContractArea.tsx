import * as React from 'react';

const ContractArea: React.FC = ({ children }) => (
  <div style={{ maxHeight: '70vh' }} className='overflow-auto'>
    {children}
  </div>
);

export default ContractArea;
