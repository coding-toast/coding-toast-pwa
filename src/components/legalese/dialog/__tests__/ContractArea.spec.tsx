import * as React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import ContractArea from '../ContractArea';

describe('<ContractArea />', () => {
  it('renders correctly', () => {
    const component = shallow(
      <ContractArea>
        <p>Hello test world!</p>
      </ContractArea>
    );

    expect(toJson(component)).toMatchSnapshot();
  });
});
