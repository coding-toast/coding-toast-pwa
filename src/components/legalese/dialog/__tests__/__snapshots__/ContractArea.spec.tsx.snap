// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ContractArea /> renders correctly 1`] = `
<div
  className="overflow-auto"
  style={
    Object {
      "maxHeight": "70vh",
    }
  }
>
  <p>
    Hello test world!
  </p>
</div>
`;
